README.rst

```diff
@@ -24,23 +24,31 @@ repository:
     **Deprecated**
     Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7.
 
-`readthedocs/build:6.0`
+`readthedocs/build:5.0`
     ``stable``
-    Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8 and PyPy3.5-7.0.0.
+    Ubuntu 18.04 supporting Python 2.7, 3.6, 3.7 and pypy3.5-7.0.0.
     This is the **stable** image supported by Read the Docs.
 
-`readthedocs/build:7.0`
+`readthedocs/build:6.0`
     ``latest``
     Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8 and PyPy3.5-7.0.0.
     This is the **latest** default image used for documentation builds and supported by Read the Docs.
 
-`readthedocs/build:8.0`
+`readthedocs/build:7.0`
     ``testing``
-    Ubuntu 20.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8, 3.9.0rc1 and PyPy3.5-7.0.0.
+    Ubuntu 18.04 supporting Python 2.7, 3.5, 3.6, 3.7, 3.8, 3.9, and PyPy3.5-7.0.0.
     Available for public usage as **testing** image. You should expect some breaking changes here.
 
 .. _readthedocs/build: https://hub.docker.com/r/readthedocs/build/
 
+.. warning::
+
+   We are transitioning to a new way of building our Docker images, see
+   `the relevant design document <https://docs.readthedocs.io/en/stable/development/design/build-images.html>`_
```

Comment:
Currently, we're only at the design doc phase, and have not added anything to our roadmap. In general, I don't think we should use design docs to communicate outside the core team, this information is a bit confusing for our users because it's very heavy on technical discussion. Once we add this work to our roadmap, we should have something here about the changes and deprecation policy however.